Source/Paths/CoordinateParser.cs

```diff
@@ -274,7 +274,7 @@ public bool TryGetFloat(out float result)
                         break;
                 }
 
-                if (_newState == NumState.invalid)
+                if (_currState == NumState.separator && _newState == NumState.invalid)
```

Comment:
I think this block should be deleted.

And you add the following condition in line 283:
```cs
if (_currState != NumState.separator && _newState < _currState)
```

Reply:
> I think this block should be deleted.
> 
> And you add the following condition in line 283:
> 
> ```cs
> if (_currState != NumState.separator && _newState < _currState)
> ```

@H1Gdev Yes this works. Updated the PR